src/applications/applications.service.ts

```
@@ -71,4 +73,32 @@ export class ApplicationsService {
     return application;
   }
+
+  async getApplicationsByUser(userId: string): Promise<Application[]> {
+    return this.applicationRepository.find({
+      where: { user: { id: userId } }, // id is string here, matching entity
+      relations: ['job'],
+      order: { createdAt: 'DESC' },
+    });
+  }
+
+  async findApplicationsByJobId(jobId: number, recruiterId: number) {
```

ESLint

[error] 'recruiterId' is defined but never used.

(@typescript-eslint/no-unused-vars)